test/integration/recommend-page.js

```diff
@@ -35,17 +36,35 @@ describe('Recommend page', () => {
 
   describe('return to Choices services', () => {
     it('the breadcrumb should have a link back to the Choices \'Services near you\'', async () => {
-      const res = await chai.request(server).get(recommendRoute);
+      const res = await chai.request(server)
+        .get(recommendRoute)
+        .query({ symptoms: 'yes' });
 
+      iExpect.htmlWith200Status(res);
       const $ = cheerio.load(res.text);
 
       expect($($('div.breadcrumb a')[1]).attr('href'))
         .to.equal('https://www.nhs.uk/service-search');
     });
 
+    it('the page should have a link to the Location search page', async () => {
```

Comment:
This test doesn't sit well within the `return to Choices services` block. Maybe just move it outside and below the title tests?

Reply:
Updated this.